File: backend/src/main/java/com/expensesplitter/settlement/dto/SettlementSummaryResponse.java
package com.expensesplitter.settlement.dto;

import lombok.*;

import java.math.BigDecimal;

@Getter
@Setter
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class SettlementSummaryResponse {

    private Long fromUserId;
    private String fromUserName;

    private Long toUserId;
    private String toUserName;

    private BigDecimal amount;
}

File: android/app/src/main/java/com/expensesplitter/app/model/settlement/SettlementSummaryResponse.kt
package com.expensesplitter.app.model.settlement

import java.math.BigDecimal

data class SettlementSummaryResponse(
    val fromUserId: Long,
    val fromUserName: String,
    val toUserId: Long,
    val toUserName: String,
    val amount: BigDecimal
)
